WordPress/src/main/java/org/wordpress/android/ui/posts/editor/StorePostViewModel.kt

```diff
@@ -90,6 +92,11 @@ class StorePostViewModel
         editPostRepository: EditPostRepository,
         site: SiteModel,
     ): ActivityFinishState {
+
+        if (!syncPublishingFeatureConfig.isEnabled()) {
```

Comment:
Added the FF here trying to have both sync and async behaviour depending on FF value.